### PR TITLE
Docs - updates per new dashboard CI/CD features

### PR DESCRIPTION
**Pull request by skierkowski**
## What did you implement

This updates the docs for CI/CD in Dashboard with a few key changes: (1) refactoring a few sections into dedicated pages, (2) updating the format of the CI/CD readme, and (3) adding content for new features.

Making this a draft PR as some of the features described are not yet available.

## How can we verify it

A preview of the docs can be seen [here](https://5df5635489037400084fb4d7--serverless.netlify.com/framework/docs/dashboard/cicd/).

### CODE REVIEW

File: docs/dashboard/cicd/README.md [hunk 1]
```diff
@@ -19,129 +19,47 @@ Serverless CI/CD enables you to automatically test and deploy services from Gith
...
+3. Add a new stage, name it “dev” and select the “default” profile.
 
-To add a new branch deployment, select the Github branch containing the `serverless.yml` and the target stage and click "Add". You must click "save settings" at the bottom of the form before the branch deployment is enabled.
+You can add more stages and profiles later.
 
-### Add a stage to a branch deployment
+### Step 2: Connect to Github
 
-If you are a first time user, the stage list may be empty. You must first [create a new deployment profile](/framework/docs/dashboard/profiles#creating-a-new-deployment-profile), and [add the deployment profile to a stage in your application](/framework/docs/dashboard/profiles#add-a-deployment-profile-to-your-application-and-stage). As mentioned in the requirements section, the deployment profile must also have an [AWS Access Role](/framework/docs/dashboard/access-roles/).
+1. Login to https://dashboard.serverless.com/
```
Comment (pmuens) [inline docs/dashboard/cicd/README.md]: We should wrap this into `[]()` so that users can click on it.

File: docs/dashboard/cicd/README.md [hunk 2]
```diff
@@ -19,129 +19,47 @@ Serverless CI/CD enables you to automatically test and deploy services from Gith
...
 
-1. Select the Github repository from the "repository" dropdown. This must be a repository that contains one or more `serverless.yml` files.
-2. Select the "base directory" containing the `serverless.yml` file. The repository must contain at least one `serverless.yml` and `service` value must match the current service in the dashboard.
-3. Select the “region” for the deployments. Only regions supported by the Serverless Dashboard are currently listed. If you plan to deploy to another region, please reach out sales or support to request support for a new region.
+If you’ve already setup an AWS Access Role in a Profile you can skip this step.
 
-## Branch Deployments
+1. To setup the AWS Access Role, login to https://dashboard.serverless.com/ and navigate to the “profiles” tab.
```
Comment (pmuens) [inline docs/dashboard/cicd/README.md]: We should wrap this into `[]()` so that users can click on it.

File: docs/dashboard/cicd/faq.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+Yes! You can use deployment profiles to add as many AWS Accounts as you would like, and map them to individual stages in
+your application.
+
+## Is it just for Serverless Framework?
+
+Yes! Serverless CI/CD is designed around the Serverless Framework to provide a seamless experience for developers.
+Anything you can deploy with the Serverless Framework you can deploy with Serverless CI/CD. The Serverless Framework is
+extensible with Plugins , so it works with a broad range of services.
+
+## Is Serverless CI/CD also available for Enterprise?
```
Comment (pmuens) [inline docs/dashboard/cicd/faq.md]: Which "Enterprise"? Are we talking about GitHub Enterprise here? (I assume we do).

File: docs/dashboard/cicd/mono-repo.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+<!--
+title: Serverless Dashboard - CI/CD Mono Repos
+menuText: Mono Repos
+menuOrder: 7
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/dashboard/cicd/faq/)
```
Comment (pmuens) [inline docs/dashboard/cicd/mono-repo.md]: This link should be updated to reflect the file name.

File: docs/dashboard/cicd/private-packages.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+<!--
+title: Serverless Dashboard - CI/CD Private Package Manager (NPM)
+menuText: Private Packages
+menuOrder: 6
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/dashboard/cicd/faq/)
```
Comment (pmuens) [inline docs/dashboard/cicd/private-packages.md]: This link should be updated to reflect the file name.

File: docs/dashboard/cicd/private-packages.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+<!-- DOCS-SITE-LINK:END -->
+
+# Using private package managers
+
+If your Serverless Framework project has a dependency on a private package manager, like NPM, you will need the CI/CD
+service to authenticate with the private package manager service.
+
+If you are using NPM follow the "[Using private packages in a CI/CD workflow](https://docs.npmjs.com/using-private-packages-in-a-ci-cd-workflow)"
```
Comment (pmuens) [inline docs/dashboard/cicd/private-packages.md]: What about Python? Or do we add this later on?

File: docs/dashboard/cicd/README.md (hunk 1 above)
Comment (skierkowski) [reply docs/dashboard/cicd/README.md]: Ahhh good call. My IDE showed it as underlined so I missed it.

**Comment by skierkowski** [comment]
@medikoo @pmuens  On the last PR I learned I need to prefix `docs: ` in my commit messages. On this PR I learned that I need to sentence-case (upper case first letter) of the message.  How can I fix that?

**Comment by medikoo** [comment]
@skierkowski if rebase is a problem do not worry, I'll just merge it via squashing, just let us know when we should do that.

**Comment by skierkowski** [comment]
@medikoo @pmuens it's ready to go live now. Can you do the rebasing? I haven't done that before. In the future I can do that, but would rather practice on a personal repo first; don't want to screw anything up here.

**Pull request merged**